### ISSUE TITLE
Custom constraints produces server error

### ISSUE THREAD

**Issue by finger563** [webgme/webgme#939]
When I try to run constraint checker on a model with a single custom constraint defined in the `meta` I get the following error output (on the server):

```
2016-04-13T15:13:53.657Z - error: [gme:server:worker:simpleworker:pid_10674] sending message failed 
{ metadata: { pid: 10674, type: 'result', error: null, result: [ [Object] ] },
  e: {} }
```

Note that no output or any message of any sort appears on the client.  

[project.json.zip](https://github.com/webgme/webgme/files/217403/project.json.zip)

The model for reference is attached.  The custom constraint (`nameConstraint`) is defined on the `Project`.

**Comment by finger563** [comment]
Does this include the issue of actually defining / saving custom constraints?  E.g. the issues with saving custom constraints that actually use the `core` object?

**Comment by pmeijer** [comment]
Yes the PR does. It turned out it had nothing to do with the using the core or not, but that the regex checking for invalid characters (e.g. `.` and `$` of the constraint name was accidentally applied to the script).

**Comment by finger563** [comment]
Ah I see, thanks!

**Comment by pmeijer** [comment]
Closed by #944
